handler/handler.go

```diff
@@ -244,11 +273,81 @@ func createOuputFolder(request *GenerateTemplateRequest) error {
 	if err != nil {
 		return err
 	}
-
+		
 	return nil
 
 }
 
+func createRepo(request *GenerateTemplateRequest){
+	repo_name := request.AppName
+	token := os.Getenv("GITHUB_AUTH_TOKEN")	
+	
+	url := "https://api.github.com/orgs/gophers-prop/repos"
+
+	payload := strings.NewReader("{\"name\":\""+repo_name+"\"}")
+
+	req, _ := http.NewRequest("POST", url, payload)
+
+	req.Header.Add("authorization", "token "+token)
+	req.Header.Add("content-type", "application/json")
+	req.Header.Add("cache-control", "no-cache")
+	req.Header.Add("postman-token", "e1c5b0b0-a573-98dd-008b-72250586c558")
+
+	res, _ := http.DefaultClient.Do(req)
+
+	defer res.Body.Close()
+	body, _ := ioutil.ReadAll(res.Body)
+
+	fmt.Println(res)
+	fmt.Println(string(body))
+
+	//fmt.Printf("Successfully created new repo: %v\n", repo.GetName())
+
+	cmd := exec.Command("git", "init", request.outputFolder)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+
+	if err := cmd.Run(); err != nil {
+		fmt.Println("Git init err :", err)
+	}
+
+	cmd = exec.Command("git", "add", "*")
+	cmd.Dir = request.outputFolder
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+
+	if err := cmd.Run(); err != nil {
+		fmt.Println("Git add err :", err)
+	}
+
+	cmd = exec.Command("git", "commit", "-m", "Initail Commit")
+	cmd.Dir = request.outputFolder
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+
+	if err := cmd.Run(); err != nil {
+		fmt.Println("Git commit err :", err)
+	}
+
+	cmd = exec.Command("git", "remote", "add", "origin", "https://github.com/gophers-prop/"+request.AppName)
```

Comment:
I would suggest to use something more with appname here. Maybe ask user to provide a repo name and validate that name with existing test repo's that are already present  or add a random 4-6 digit no with appname .